packages/esm-cervical-cancer-app/package.json

```diff
@@ -77,7 +77,7 @@
     "@babel/preset-typescript": "^7.3.3",
     "@babel/runtime": "^7.17.8",
     "@openmrs/esm-framework": "^3.2.1-pre.1055",
-    "@openmrs/esm-patient-common-lib": "^3.1.1-pre.1346",
+    "@openmrs/esm-patient-common-lib": "next",
```

Comment:
made this change cause there seems to be a variation on how the community names releases. The pre release numbers aren't consistent and doesn't follow the normal numerical format and that's why we were having issues picking the new changes from this library. 


<img width="838" alt="Screenshot 2022-06-02 at 13 04 34" src="https://user-images.githubusercontent.com/15266028/171606715-929bae86-081a-43ea-afcd-9c044f3593a8.png">